src/core/Colors/Colors.baseStyles.tsx

```diff
@@ -17,20 +17,39 @@ export const baseStyles = withSuomifiTheme(
       flex-direction: column;
       justify-content: flex-end;
       float: left;
-      width: 192px;
-      height: 180px;
+      width: 208px;
```

Comment:
Where do these magic numbers come from? Looks like these were just updated, but this kind of arbitrary looking styles based on absolute values would benefit from comments in my opinion. These are used with styleguidist?

Reply:
These values for `width` and `height` was updated to keep the visual presentation, in the styleguidist, like it was previously; looking square.

Possibly later this could be done dynamically based on available size, but at this point this it not worth the effort.

Reply:
If something was done regarding the visual presentation, I think a screenshot or two would make the review a lot easier, even if they are just to show that things stayed the same, like in this case.

Reply:
I didn't add any screenshots for the PR as there were no visual changes needed to be done with this fix. And change to these values here were done only to keep the same appearance as before.

But if there will be later changes that will change the color tokens presentation to be more dynamic, screenshots would be very much needed to show the difference eg. in some example cases (_different browser width_).